### DISCUSSION TITLE
v2.7.0 Agent issue - Take Control results in white browser window

### DISCUSSION THREAD

**Discussion by MtTime73** [amidaware/tacticalrmm#1854] [General]
**Server Info (please complete the following information):**
 - OS: Ubuntu 22.04.4 LTS
 - Browser: Brave 
 - RMM Version (as shown in top left of web UI): v0.18.2

**Installation Method:**
  - [x] Standard
  - [ ] Standard with `--insecure` flag at install
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 2.7.0
- Agent OS: Win 11 Pro, version 23H2 (downloading and installing as I write this)

**Describe the bug**
When issuing the Take Control command, the resulting MeshCentral browser window has a white box where the remote session used to be.

**To Reproduce**
1. Go to TRMM admin panel
2. Click on agent
3. Right click on agent and select Take Control

**Expected behavior**
A browser window (that's black) with Connect / RDP Connect buttons, Disconnected status, Action / Settings buttons, Rotate Right, Rotate Left, and Toggle View Mode icons.

**Screenshots**
![brave_AKWyx1Hjoi](https://github.com/amidaware/tacticalrmm/assets/58580428/e67ff002-1c44-4643-87b6-c01796a74d09)

**Additional context**
I have a bunch of agents running v2.6.2 and all of them work (aka Take Control works as expected). I am also using Brave browser to connect to the v2.6.2 agents.

Thanks!

**Comment by dinger1986** [comment]
is meshagent running on those devices?

**Reply by MtTime73** [reply]
I've downloaded the installer same as I always have - and ran on the workstation.  

Services does show the Mesh Agent is running.

**Reply by dinger1986** [reply]
does remote background work? does it show up in mesh.yourdomain.com?

**Reply by MtTime73** [reply]
Is there a way to install the older version of the agent for now?

The remote background is still not working using Take Control (still showing as a white background).

I've uninstalled the agent on the endpoint (via add/remove programs, agent removal showed successful, and then uninstalled from the console), and did a reboot.

I've re-downloaded the agent from the console and installed (run as admin), agent installs successfully, then shows up in the console within seconds. All is good at this point.

Head over to mesh.mydomain.com and the endpoint is not showing up.

To verify, I confirmed this behavior now in both Chrome (v124.0.6367.61) and Brave (v1.65.114 Chromium: 124.0.6367.60)

**Reply by MtTime73** [reply]
I did upgrade to v0.18.2 recently. 

Should I possibly update/upgrade the server (if applicable) again and then run the script with the --force switch?

**Reply by dinger1986** [reply]
Did you upgrade the server os? If so from what?

It could be the new sync feature https://docs.tacticalrmm.com/mesh_integration/#turn-off-the-sync-feature

How many agents do you have? Do you sponsor?

**Reply by MtTime73** [reply]
I had seen that post about the new sync feature, but without fully understanding it's implications I didn't want to uncheck. I have now unchecked and voila, its fixed!

I do sponsor but I believe I'm at a $25/mn sponsorship. I mainly use for a home lab setup but acknowledge how much I rely on it. I've been supporting for awhile now, but just pulled up the AmidaWare Inc. sponsor page on Github and plan on moving to the $50/mn sponsorship.

In case this isn't said enough: I love the product, commend the entire team on everything that's been developed and built, and thanks for providing such great support!

**Reply by dinger1986** [reply]
glad its working, it would be good to understand why it didnt work, if you have time can you read through nhttps://github.com/amidaware/tacticalrmm/issues/1814 and see if theres anything there which could be the issue
